CHANGELOG.md

```diff
@@ -3,6 +3,32 @@ Changelog information can also be found in each release's git tag (which can be
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v1.0.0-beta.6]
```

Comment:
This is written as a link but won't link anywhere. The actual links are at the bottom of the file, for example on line 358 (of this PR): 
```md
[v1.0.0-beta.2]: https://github.com/bitcoindevkit/bdk-ffi/compare/v1.0.0-alpha.11...v1.0.0-beta.2
```

They link to the list of commits comparison between the two tags (note that we have also forgotten to do this for beta 5... we might want to fix both in this commit?)

Reply:
thanks for catching this, fixed (for both beta 5 and beta 6!), added the two lines needed around line 358.